mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -125,8 +125,7 @@ def __init__(self, target: str):
             self.machine = 'arm'
         else:
             self.machine = target
-        if mesonlib.version_compare(self.version, '>=19.29.29917'):
-            self.base_options.add(mesonlib.OptionKey('b_sanitize'))
+        self.base_options.add(mesonlib.OptionKey('b_sanitize'))
```

Comment:
This is a behavioural change. The option should not be added at all if the compiler does not support it. It should not handle it if it exists because it conceivably might come from a different compiler. So the version check and warning below is correct but this one is not.